src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -1,244 +1,487 @@
-
-Relation to Programmatic Work
+    handled in the following week by a new **Primary/Backup** pair.
+  * If customer indicates immediate response is required, the **Primary**
+    should query the whole team to arrange another developer who can
+    handle it.
+
+.. _sre_housekeeping:
+
+SRE vs. Product Development
+---------------------------
+
+Part of the reason for formalizing this process is the recognition of a
+different category of work,
+`Site Reliability Engineering <https://en.wikipedia.org/wiki/Site_Reliability_Engineering>`_ (SRE),
+that is an essential part of maintaining the overall quality of
+a software product as well as the productivity of both developers and users
+of the software alike. Nonetheless, SRE work is very different from
+conventional *product development* type of work where
+bug fixes, technology refreshes, feature enhancements and major releases are
+prioritized, planned and resources assigned to hit target deliverable
+dates.
+
+Small, easily fixed issues that impact one user's productivity often impact
+others. Likewise for developer productivity issues. Often, these kinds of
+issues can wind up falling through the cracks of traditional software project
+management and planning processes. However, such issues also often represent
+low cost high benefit improvements in quality of either the software itself or
+the development or deployment processes supporting it. We refer to issues of
+this nature as general *housekeeping* or *low-hanging fruit* type issues.
+
+Apart from acknowledging their existence, a key part of this process is the
+allocation of resources for the sole purpose of supporting SRE activities
+and developing a practice of continuously crafting small corrective actions
+arising from SRE inquiries.
+
+Consequently, another key role of the **Primary** is to use any time not working
+active SRE issues to fix other *low-hanging fruit* issues from the
+*product development* backlog. As a rule of thumb, low-hanging
+fruit is considered to be anything that the **Primary** believes is fixable
+within a half-day's (4 hours) worth of effort. When there are many such tasks in
+the system to work on, the **Primary** is free to use his/her judgment to decide
+which s/he can most productively address.
+
+Part of the acknowledgment of this new category of work is the new
+`issue tracker <https://github.com/visit-dav/live-customer-response/issues>`_
+for tracking it. *New* SRE activity will start with an issue being added there.
+As an SRE incident unfolds it may result in either the same issue being moved to the
+`product development <https://github.com/visit-dav/visit/issues>`_ issue tracker
+and/or new issue(s) being added to the 
+`product development <https://github.com/visit-dav/visit/issues>`_ tracker.
+Any new *product development* issues should be linked back to the original
+SRE issue that spawned them.
+
+Finally, there are likely a number of
```

Comment:
I'm not sure that this paragraph should be here. Maybe this should just be a ticket for us to address. It's relation to the SRE process is temporary; once we audit the issues, the task is done and the docs would need to be updated.

Reply:
Yeah, I kinda agree. I was on the fence about keeping it there. I did already add a ticket though.